shuffle/move/examples/Message/sources/set_message.move

```diff
@@ -0,0 +1,7 @@
+script {
+    use MessageAddress::Message;
+
+    fun set_message(account: signer, message: vector<u8>) {
```

Comment:
Can we not use script funs for this?

Reply:
We want to, but there are some limitations to Script Functions and the generated transaction builders. This was the stopgap.